house/views.py

```diff
@@ -1,13 +1,13 @@
+from house.constants import MINE_ADD_EXPENSE_ROUTE
 from django.contrib.auth.decorators import login_required
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
-
 from expenses.models import Expenses
-from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE, GLOBAL_PAGE_CITY_DROPDOWN_ROUTE
-from house.constants import MINE_MINE_PAGE_ROUTE, HOUSE_CREATE_ROUTE
-from .forms import HouseForm, HouseCreationForm
-from .helpers import _filter_houses_by_form
+from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE, GLOBAL_PAGE_CITY_DROPDOWN_ROUTE, \
+ MINE_PAGE_ROUTE, HOUSE_CREATE_ROUTE
+from .forms import HouseForm, HouseCreationForm, ExpenseForm
```

Comment:
You have line 1 which is also in house.constants 
`from house.constants import (
 MINE_ADD_EXPENSE_ROUTE, HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE, GLOBAL_PAGE_CITY_DROPDOWN_ROUTE, 
 MINE_PAGE_ROUTE, HOUSE_CREATE_ROUTE
)`

Reply:
fixed